docs/intro/upcoming-events.md

```diff
@@ -0,0 +1,20 @@
+---
+title: Upcoming Events
+hide_title: true
+slug: /upcoming-events
+---
+
+<img src="/img/title/monthly-update.svg" className="titlePic" />
+
+
+### April 5, 2023 | Office Hours
```

Comment:
Maybe it's worth noting somewhere that office hours will take place on first Wednesday every month.